packages/devtools_app/lib/src/flutter/table.dart

```diff
@@ -1,6 +1,7 @@
 import 'dart:math';
 
 import 'package:flutter/material.dart';
+import 'package:flutter/src/scheduler/ticker.dart';
```

Comment:
I imagine the auto-imports suggested this bad import of a src import of the scheduler library rather than
flutter/scheduler.dart
Fix the import and maybe file a bug.